docs/source/library/Synthesis.rst

```diff
@@ -0,0 +1,20 @@
+Quantum Circuit Synthesis
+===========================
+
+The main functionality for synthesizing quantum circuits is the :func:`~mqt.qmap.synthesize_clifford` and the :func:`~mqt.qmap.optimize_clifford` method.
+
+
+
+    .. currentmodule:: mqt.qmap
+    .. automethod:: mqt.qmap::synthesize_clifford
+    .. automethod:: mqt.qmap::optimize_clifford
```

Comment:
For some reason, the method signatures seem to be a little messed up (some parameters are missing).
Furthermore, it would be great to link to the `SynthesisConfiguration` in the docstrings (`bindings.cpp`)

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/6358767/207912112-7d388cc2-efe0-4633-9491-07e8bc258d76.png">
Still broken

Reply:
I'm still trying to find out why that doesn't work, sometimes it does it correctly locally, but I have yet to find out why..